File: refactored/duplicate-assert/python-pytest/FileLengthCheckTest_testExtensions.py
import pytest

class TestFileLengthCheck:

    def test_extensions_1(self):
        check = FileLengthCheck()
        check.set_file_extensions("java")
        assert check.get_file_extensions()[0] == ".java", "extension should be the same"

        with pytest.raises(IllegalArgumentException) as excinfo:
            check.set_file_extensions(None)
        assert str(excinfo.value) == "Extensions array can not be null", "Invalid exception message"

    def test_extensions_2(self):
        check = FileLengthCheck()
        check.set_file_extensions(".java")
        assert check.get_file_extensions()[0] == ".java", "extension should be the same"

        with pytest.raises(IllegalArgumentException) as excinfo:
            check.set_file_extensions(None)
        assert str(excinfo.value) == "Extensions array can not be null", "Invalid exception message"

File: refactored/duplicate-assert/java-junit/FileLengthCheckTest_testExtensions.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

public class FileLengthCheckTest extends AbstractModuleTestSupport {

    @Test
    public void TestExtensions1() {
        FileLengthCheck check = new FileLengthCheck();
        check.setFileExtensions("java");
        assertEquals(".java", check.getFileExtensions()[0], "extension should be the same");

        Exception exception = assertThrows(IllegalArgumentException.class, () -> check.setFileExtensions(null));
        assertEquals("Extensions array can not be null", exception.getMessage(), "Invalid exception message");
    }

    @Test
    public void TestExtensions2() {
        FileLengthCheck check = new FileLengthCheck();
        check.setFileExtensions(".java");
        assertEquals(".java", check.getFileExtensions()[0], "extension should be the same");

        Exception exception = assertThrows(IllegalArgumentException.class, () -> check.setFileExtensions(null));
        assertEquals("Extensions array can not be null", exception.getMessage(), "Invalid exception message");
    }
}
